spec/kong_tests.conf

```diff
@@ -8,6 +8,9 @@ cluster_listen_rpc = 127.0.0.1:9373
 ssl_cert = spec/fixtures/kong_spec.crt
 ssl_cert_key = spec/fixtures/kong_spec.key
 
+admin_ssl_cert = spec/fixtures/kong_spec.crt
+admin_ssl_cert_key = spec/fixtures/kong_spec.key
```

Comment:
I am testing it the same way we are testing the HTTPs proxy port. These things should be tested.